### DISCUSSION TITLE
🚀 Feature Request:能否支持一下WebStrorm

### DISCUSSION THREAD

**Discussion by LanXa7** [alovajs/alova#541] [Feature Request]
### What problem does the new feature need to solve?

WebStrorm的用户还是不少的 是否后续可以考虑增加插件呢

### How important is it to you?

Better to have

**Comment by MeetinaXD** [comment]
我们暂时还没有这样的计划，ws 插件开发使用的 Java 技术栈和我们当前的实现不兼容，重新开发需要不少的工作量。

如果当前确实有这样的需求，可以考虑下折中的方案：

仅仅使用 vscode 安装插件来自动生成类型声明。然后在偏好的 IDE 中通过 `Apis.xxx` 的全局方式来使用接口。
